src/amp-story-embed.js

```diff
@@ -31,6 +33,58 @@ const CSS = `
   main { display: flex; flex-direction: row; height: 100%; }
   .i-amphtml-story-embed-loaded iframe { opacity: 1; }`;
 
+/** @const {string} */
+const TAG = '[AMP-STORY-EMBED]';
+
+/**
+ * Logging.
+ */
+function log() {
+  const var_args = toArray(arguments);
+  var_args.unshift(TAG);
+  console /*OK*/.log
+    .apply(console, var_args);
+}
+
+/**
+ * Gets encoded url for viewer usage.
+ * @param {string} href
+ * @return {!Location}
+ */
+function getEncodedUrl(href) {
+  const params = {
+    visibilityState: 'inactive',
+    origin: parseUrlDeprecated(self.location.href).origin,
```

Comment:
If `parseUrl` is deprecated, what's the way we're supposed to do this?

Reply:
`Services.urlForDoc` has the same function. 

@jridgewell Any end in sight for this migration? :)

Reply:
Ah, we can't use services here, I think, since v0 won't be on the page.

Reply:
I'm planning on using `parseUrlWithA`, which is what `parseUrlDeprecated` uses internally https://github.com/ampproject/amphtml/blob/2ca921f9e29fb54a872a0400d781fac15bb362bf/src/url.js#L119

Reply:
If this is a non-amp page, then `parseUrlWithA` is appropriate.